bindings/python/pydeck/pydeck/_version.py

```diff
@@ -3,5 +3,6 @@
 # Copyright (c) Uber Technologies, Inc.
 # Distributed under the terms of the Modified BSD License.
 
+# TODO should this be the same version number as the deck.gl release?
```

Comment:
If we need to republish this with every deck.gl release to ensure we capture bug fixes etc it would make sense to align the version numbers.